scripts/upgrade

```diff
@@ -98,22 +119,10 @@ ynh_add_nginx_config
 # Set permissions to app files
 chown -R root: $final_path
 
-#=================================================
-# SETUP SSOWAT
-#=================================================
-ynh_script_progression --message="Upgrading SSOwat configuration..."
-
-# Make app public if necessary
-if [ $is_public -eq 1 ]
-then
-	# unprotected_uris allows SSO credentials to be passed anyway.
-	ynh_app_setting_set --app=$app --key=skipped_uris --value="/"
-fi
-
 #=================================================
 # RELOAD NGINX
 #=================================================
-ynh_script_progression --message="Reloading nginx web server..."
+ynh_script_progression --message="Reloading nginx web server..." --weight=1
```

Comment:
Just FYI, if there is no `--weight` = `--weight=1`

Reply:
Actually it does work, but it's useless. Since, without a work value, the helper assume a value of 1.